parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -71,9 +122,11 @@ installNetworkPlugin() {
 }
 
 downloadCNI() {
-    mkdir -p $CNI_DOWNLOADS_DIR
-    CNI_TGZ_TMP=${CNI_PLUGINS_URL##*/} # Use bash builtin ## to remove all chars ("*") up to the final "/"
-    retrycmd_get_tarball 120 5 "$CNI_DOWNLOADS_DIR/${CNI_TGZ_TMP}" ${CNI_PLUGINS_URL} || exit $ERR_CNI_DOWNLOAD_TIMEOUT
+    downloadDir=${1:-${CNI_DOWNLOADS_DIR}}
```

Comment:
nit: can we preserve the casing for variables? we have upper case everywhere in cse scripts

Reply:
Actually I found multiple naming convention in a single .sh file. For example, `images_to_delete`, `NEEDS_CONTAINERD`, `allMCRImages`. That's why I picked one which is
```
ALL_UPPER_CASE for constants, global variables and environment variables
camelCase for local variables.
```
We can definitely revise and unify all the variables. Does this sound make sense?

Reply:
yeah, Anuj and I were talking about conditional syntax being inconsistent and all over the place as well...

maybe would be better to do that sort of thing in a separate PR